src/main/java/org/mitre/synthea/export/rif/PDEExporter.java

```diff
@@ -206,4 +202,44 @@ private static BigDecimal getDrugOutOfPocketThreshold(int year) {
     double threshold = pdeOutOfPocketThresholds.getOrDefault(year, 4550.0);
     return BigDecimal.valueOf(threshold);
   }
+
+  private static String getResidenceCode(Person person, HealthRecord.Encounter encounter) {
+    Set<ClaimType> claimTypes = RIFExporter.getClaimTypes(encounter);
+    String residenceCode = "00"; // 00=not specified
+    double roll = person.rand();
+    if (claimTypes.contains(ClaimType.SNF)) {
+      residenceCode = "03"; // 03=long-term
+    } else if (claimTypes.contains(ClaimType.HHA)) {
+      if (roll <= 0.95) {
+        residenceCode = "01"; // 01=home
+      } else {
+        residenceCode = "04"; // 04=assisted living
+      }
+    } else if (claimTypes.contains(ClaimType.HOSPICE)) {
+      residenceCode = "11"; // 11=hospice
+    } else if (claimTypes.contains(ClaimType.INPATIENT)) {
+      if (roll <= 0.95) {
+        residenceCode = "01"; // 01=home
```

Comment:
If you change this line to "03" it improves the "long term" frequency (5% in my test) without impacting the "home" value much.